src/odemis/driver/test/semcomedi_test.py

```diff
@@ -56,7 +56,9 @@
 CONFIG_CNT = {"name": "cnt", "role": "cnt", "source":0}
 CONFIG_SCANNER = {"name": "scanner", "role": "ebeam", "limits": [[-5, 5], [3, -3]],
                   "channels": [0, 1], "settle_time": 10e-6, "hfw_nomag": 10e-3,
-                  "park": [8, 8]}
+                  "park": [8, 8],
+                  "scanning_ttl": {4: True, 2: [True, "external"], 3: [False, "blanker", True]}
+                  }
```

Comment:
we could add an extra comment here about digital port mapping, or maybe it's too much just for a test file.

Reply:
Yes, for the test case, I expect that the reader should be able to read the docstring, so no need to add too much extra comments ;-)